app/components/dashboard/audit_status_component.html.erb

```diff
@@ -0,0 +1,60 @@
+<div class="card border-0 text-start">
+  <div class="card-body" />
+    <div class="card-title fs-4 fw-semibold d-flex justify-content-between">
+        Audit&nbsp;&nbsp;
```

Comment:
Consider adding an `ms-2` class to the span instead of spacing with `&nbsp;`

Reply:
thanks - I used `pe` but same diff.